### DISCUSSION TITLE
Is there a way to use OpenApiHono with the handle of hono/aws-lambda?

### DISCUSSION THREAD

**Discussion by albertovasquez** [honojs/hono#3591] [General]
Something like this?

```
import { OpenAPIHono } from "@hono/zod-openapi";
import { handle } from "hono/aws-lambda";

const app = new OpenAPIHono();
return handle(app);
```

**Comment by maou-shonen** [comment]
I'm not using AWS Lambda,
but if you encounter an error, I think this might work.

```ts
const oapiApp = new OpenAPIHono()
const app = new Hono()

oapiApp.doc('/openapi.json', {
  ...your openapi schema
})

app.get('/openapi.json', (c) => oapiApp.request('/openapi.json'))

export default app
```

**Reply by albertovasquez** [reply]
Thank you.  I wanted to see how to get the context and event from aws lambda my hono c

**Comment by MathurAditya724** [comment]
I am working on this middleware for OpenAPI support in Hono: https://github.com/rhinobase/hono-openapi. It's just a middleware, so if you have an AWS handler-based Hono app, this should work like normal.
